src/components/Participant/UpdateParticipant/UpdateParticipant.tsx

```diff
@@ -439,7 +440,8 @@ const UpdateParticipant = () => {
           updateCancelState: true,
         };
       }
-      case "Sex": {
+      case "Sex":
```

Comment:
Should this be capitalised and the other one isn't?  Seems strange.  Also have you looked at the percentages of the "Start registration" stepper bar?